x/lockup/types/msgs.go

```diff
@@ -111,8 +111,8 @@ func (m MsgBeginUnlocking) ValidateBasic() error {
 		return fmt.Errorf("invalid lockup ID, got %v", m.ID)
 	}
 
-	// only allow unlocks with a single denom
-	if m.Coins.Len() != 1 {
+	// only allow unlocks with a single denom or empty
+	if m.Coins.Len() != 1 || !m.Coins.Empty() {
```

Comment:
Doesnot 'empty' just mean `coins.len() == 0`?

Reply:
yep, I am still working on this because the next check is also incorrect